libdevcore/CommonData.cpp

```diff
@@ -93,6 +93,7 @@ bool dev::passesAddressChecksum(string const& _str, bool _strict)
 string dev::getChecksummedAddress(string const& _addr)
 {
 	string s = _addr.substr(0, 2) == "0x" ? _addr.substr(2) : _addr;
+	boost::erase_all(s, "_");
```

Comment:
I have the feeling that CommonData.cpp should not be concerned about underscores. Can you move the "cleanup" to the call site in AST.cpp?

Reply:
I agree with that and I was close to removing it. Let me do that then.